# Translation of clouderp Server.
# This file contains the translation of the following modules:
# * im_clouderp_support
# 
# Translators:
# Mateo Tibaquirá <nestormateo@gmail.com>, 2015
msgid ""
msgstr ""
"Project-Id-Version: clouderp 9.0\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2015-09-29 10:26+0000\n"
"PO-Revision-Date: 2015-10-28 13:24+0000\n"
"Last-Translator: Mateo Tibaquirá <nestormateo@gmail.com>\n"
"Language-Team: Spanish (Colombia) (http://www.transifex.com/clouderp/clouderp-9/language/es_CO/)\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: \n"
"Language: es_CO\n"
"Plural-Forms: nplurals=2; plural=(n != 1);\n"

#. module: im_clouderp_support
#. gce-web
#: code:addons/im_clouderp_support/static/src/js/im_clouderp_support.js:76
#, python-format
msgid "Error"
msgstr "Error"

#. module: im_clouderp_support
#. gce-web
#: code:addons/im_clouderp_support/static/src/js/im_clouderp_support.js:76
#, python-format
msgid ""
"The connection with the clouderp Support Server failed. Please retry in a few "
"minutes, or send an email to support@golive.pt ."
msgstr "La conexión con el Servidor de Soporte de clouderp falló. Por favor vuelva a intentarlo en unos minutos, o envíe un correo a support@golive.pt."
